Default to contents for project show page

hectorcorrea opened the pull request:
Switched the show page to show the contents and the metadata to be the details page

Closes #1110

hectorcorrea commented on app/views/projects/show.html.erb:
```diff
@@ -1,117 +1,56 @@
-  projectStyle('<%= @project_session %>');
-  projectTab();
+    projectStyle('<%= @project_session %>');
+    projectTab("<%= project_path %>", "<%= project_details_path %>");
```
@JaymeeH I changed the JavaScript to pass the URLs that Rails generates so that the JavaScript in `projectTab()` does not have to guess what's the correct URLs for contents and details.

hectorcorrea commented on app/views/projects/details.html.erb:
```diff
@@ -0,0 +1,117 @@
+  // Make the AJAX call to fetch the aterm script
+  showCreateScript('<%= project_create_script_path(format: "json") %>');
+  projectStyle('<%= @project_session %>');
+  projectTab("<%= project_path %>", "<%= project_details_path %>");
```
@JaymeeH I changed the JavaScript to pass the URLs that Rails generates so that the JavaScript in `projectTab()` does not have to guess what's the correct URLs for contents and details.

JaymeeH reviewed (approved):
Overall looks good, I like the change of passing the url directly to the js. One little thing i noticed is the latest downloads panel on the dashboard is causing the quota to display on a new line. Can we make it so that the container width always fits whats inside? I'm also ok with merging and addressing that in a separate ticket, its up to you.
![Screenshot 2024-12-13 at 11 23 16 AM](https://github.com/user-attachments/assets/79969b7c-11a0-49b3-96cb-5117ebf82cf3)

hectorcorrea commented:
@JaymeeH I opened a new issue for the display https://github.com/pulibrary/tigerdata-app/issues/1129. Good catch, I think we introduced that when we added the sidebar.
